ui/00_message/achievement/achievement_info.xml

```diff
@@ -6615,17 +6623,17 @@ level 30 in crafting</translation>
   <message>
     <key>ACHIEVEMENT_INFO_2531</key>
     <original>パートナーポーンの好感度をより上げた</original>
-    <translation>Raise more the affection of the Partner Pawn</translation>
+    <translation>Raise the affection of the Partner Pawn</translation>
```

Comment:
As painful as this line is, it needs to distinguish itself from the other achievement with a similar line.

`ACHIEVEMENT_INFO_2313`, `ACHIEVEMENT_INFO_2314`, `ACHIEVEMENT_INFO_2315`

Either revert the changes to these three lines or find a better way to say it.